src/NotificationCenter.php

```diff
@@ -362,10 +362,11 @@ public function createBasicStampsForNotification(int $id, TokenCollection|array
 
         $stamps = new StampCollection([new NotificationConfigStamp($notificationConfig)]);
 
-        if (null === $locale && ($request = $this->requestStack->getCurrentRequest()) instanceof Request) {
-            $stamps = $stamps
-                ->with(new LocaleStamp(LocaleUtil::formatAsLocale($request->getLocale())))
-            ;
+        if (
+            null !== $locale
+            || (($request = $this->requestStack->getCurrentRequest()) instanceof Request && $locale = $request->getLocale())
```

Comment:
Can you please put this logic outside the `if()` statement to make it more readable? I don't like nesting variables in if statements that much.

Reply:
see 99029c07135bffdb53054905039c58d44f251878